### ISSUE TITLE
Unable to add games DSTT flashcard

### ISSUE THREAD

**Issue by MrDemocracy** [Robz8/TWLoader#465]
### Your 3DS

Tell us which console do you have:

- [ ]  Old 3DS
- [ ]  Old 3DS XL
- [x]  2DS
- [ ]  New 3DS
- [ ]  New 3DS XL
- [ ]  New 2DS XL

### CFW and configuration

Tell us which CFW do you have:

- [x]  Boot9Strap (SigHax)
- [ ]  A9LH (Only Sysnand)
- [ ]  A9LH + Emunand
- [ ]  Luma3DS
- [ ]  AuReiNAND
- [ ]  Other CFW [specify]
- [ ]  Older CFW
- [ ]  Not using CFW / Using TWLoader in 3DSX format

### Console version

Tell us which version do you have:

- [x]  >= 11.4
- [ ]  < 11.4

### TWLoader version

Tell us which versions you are using:

- [x]  Latest release
- [ ]  I've compiled myself _[insert here commit hash]_
- [ ]  Old release

### Expected behaviour

After selecting `Add Games`, waiting for the DSTT menu to load and starting a game I expected it to add the game.

### Actual behaviour

It didn't add the game.

### Steps to reproduce

1. `Add Games`
2. Select the rom
3. Close the game (3ds home button)
4. Open TWLoader

Here's my `settings.ini`:
```
[FRONTEND]
FIRST_TIME_MSG_VIEWED = 1
PAGE_NUMBER = 0
QUICK_START = 0
LANGUAGE = -1
THEME = 0
SUB_THEME = 0
COLOR = 0
MENU_COLOR = 0
WOOD_ICONSCALEEFFECT = 1
SHOW_FILENAME = 0
TOP_BORDER = 1
ICON_SIZE = 0
COUNTER = 0
CUSTOM_BOTTOM = 0
TOP_LAYOUT = 0
FILETYPE = 0
SHOW_BOOT_ANIMATION = 2
BOOT_ANIMATION = 0
HEALTH&SAFETY_MSG = 1
AUTOUPDATE = 0
AUTODOWNLOAD = 0
ROM_FOLDER = roms/nds
FCROM_FOLDER = roms/flashcard/nds
GBROM_FOLDER = roms/gb
NESROM_FOLDER = roms/nes
NAME = Test

[TWL-MODE]
RAINBOW_LED = 0
TWL_CLOCK = 0
SOUND_FREQ = 0
LOCK_ARM9_SCFG_EXT = 0
RESET_SLOT1 = 0
SLOT1_ENABLESD = 0
ROM_TYPE = 0
FORWARDER = 1
FLASHCARD = 0
BOOTSTRAP_FILE = 0
DEBUG = -1
LAUNCH_SLOT1 = 1
SLOT1_KEEPSD = 0
GBARUNNER = 0
GAMESETTINGS_PATH = sd:/_nds/twloader/gamesettings/flashcard/_nds/twloader.nds
```

**Comment by Robz8** [comment]
You forgot to extract what's in the `flashcardsdroot (all)` in TWLoader.7z, to the flashcard's microSD root.

**Comment by MrDemocracy** [comment]
No, I have done this process: https://github.com/Robz8/TWLoader/wiki/Rom-loading#rom-loading-from-flashcardslot-1
